package.json

```diff
@@ -1,133 +1,44 @@
-        "pre-commit": "lint-staged",
-        "lint": "eslint --ignore-path .gitignore ."
-    },
-    "lint-staged": {
-        "src/**/*.{js,jsx}": [
-            "eslint --max-warnings=0",
-            "prettier --write",
-            "jscpd --reporters consoleFull -t 10"
-        ],
-        "src/**/*.{json,css,scss,md}": [
-            "prettier --write"
-        ]
-    },
-    "prettier": {
-        "printWidth": 120,
-        "bracketSpacing": false,
-        "tabWidth": 4,
-        "singleQuote": true,
-        "jsxBracketSameLine": true,
-        "jsxSingleQuote": true
-    },
-    "eslintConfig": {
-        "settings": {
-            "import/resolver": {
-                "node": {
-                    "extensions": [
-                        ".js",
-                        ".jsx"
-                    ],
-                    "moduleDirectory": [
-                        "src",
-                        "node_modules"
-                    ]
-                }
-            }
-        },
-        "extends": [
-            "react-app",
-            "airbnb",
-            "prettier"
-        ],
-        "rules": {
-            "no-underscore-dangle": "off",
-            "import/no-cycle": "off",
-            "import/prefer-default-export": "off",
-            "no-class-assign": "off",
-            "no-plusplus": "off",
-            "jsx-a11y/anchor-is-valid": "off",
-            "jsx-a11y/no-static-element-interactions": "off",
-            "jsx-a11y/click-events-have-key-events": "off",
-            "react/jsx-props-no-spreading": "off"
-        }
-    },
-    "eslintIgnore": [],
-    "browserslist": {
-        "production": [
-            ">0.2%",
-            "not op_mini all"
-        ],
-        "development": [
-            "last 1 chrome version",
-            "last 1 firefox version",
-            "last 1 safari version"
-        ]
-    }
+  "name": "create-react-app-with-typescript",
+  "version": "5.0.0",
+  "private": true,
+  "proxy": "http://localhost:9999/",
+  "dependencies": {
+    "@emotion/react": "^11.9.3",
+    "@emotion/styled": "^11.9.3",
+    "@mui/icons-material": "^5.8.4",
+    "@mui/material": "^5.9.2",
+    "@mui/x-data-grid": "^5.14.0",
+    "@types/react": "18.0.15",
+    "@types/react-dom": "^18.0.6",
+    "axios": "^0.27.2",
+    "cookie-parser": "^1.4.6",
+    "cors": "^2.8.5",
+    "http-proxy-middleware": "^2.0.6",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^6.3.0",
+    "react-scripts": "^5.0.1",
+    "recharts": "^2.1.13",
+    "typescript": "^4.7.4",
+    "yup": "^0.32.11"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
```

Comment:
are all of the supported browser versions verified?

Reply:
No, these were the default values that came from the cra-with-typescript project.